### ISSUE TITLE
Graphical bug in TIPS menu

### ISSUE THREAD

**Issue by VLF100** [07th-mod/umineko-answer#34]
When clicking on a portrait of a character, sometimes the highligh doesnt disappear until you hover over them again.
Screenshots:
![20180203104124_1](https://user-images.githubusercontent.com/24494940/35766027-29400c5e-08d0-11e8-857c-c70a3a8a15d4.jpg)
Likewise, the same happens with the Execute and Resurrect buttons:
![20180203104248_1](https://user-images.githubusercontent.com/24494940/35766032-44b81832-08d0-11e8-8779-28310b61304a.jpg)
![20180203104259_1](https://user-images.githubusercontent.com/24494940/35766036-45e1a75a-08d0-11e8-944d-4def74d3e3e6.jpg)

**Comment by drojf** [comment]
So what happens is, if you click on a different character portrait (causing the game to load new text) then you move the mouse out of bounds of the character portrait while the new txt is loading, the game misses the "you moved the mouse out of the portrait" event, and doesn't un-mouseover the portrait. I did look into fixing this but i'm too afraid i'll screw something up, and i'm not sure if there's one spot I can apply the fix or if i'd have to apply it in multiple places.

In short, I don't think it's big enough a bug to warrant fixing (especially since I think I might screw something else up) so I won't fix it. Sorry : (

**Comment by VLF100** [comment]
It's fine, it is not gamebreaking. I just wanted to report so you knew about it.
